scipy/stats/contingency.py

```diff
@@ -271,3 +272,122 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+def association(arr, chi2_stat, stat="cramer"):
+    """Calculates degree of association between variables
+    that are nominal or greater.
+
+    Allows for specification of one of four related methods,
+    Tschuprow's T, Pearson's Contingency Coefficient, Cramer's V and Phi.
```

Comment:
The first line should be a concise one-line summary (see the [NumPy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#sections)).  Perhaps something like this:
```suggestion
    """Compute the measure of association of two nominal variables.

    The function provides the option for computing one of three
    measures of association of two nominal variables from the data
    given in a 2-d contingency table: Tschuprow's T, Pearson's
    contingency coefficient, and Cramer's V.
```

Reply:
done.